### PR TITLE
ENG-909: Change CIDR for rel.cd

### PR DESCRIPTION
**Pull request by Sudokamikaze**
Netmaps:
```
netMap = [:]
netMap['eu-west-1b']
```
Will be updated after approval & stack run as we don't know their new ids yet

### CODE REVIEW

**Review by delgod — APPROVED** [review]

**Review by hors — APPROVED** [review]

File: IaC/rel.cd/JenkinsStack.yml [hunk 1]
```diff
@@ -457,11 +457,11 @@ Resources:
...
         - DeviceIndex: 0
-          SubnetId: !Ref JSubnetB
+          SubnetId: !Ref JRELSubnetB
           Groups:
-          - !GetAtt JVPC.DefaultSecurityGroup
-          - !GetAtt HTTPSecurityGroup.GroupId
-          - !GetAtt SSHSecurityGroup.GroupId
+          - !GetAtt JRELVPC.DefaultSecurityGroup
+          - !GetAtt HTTPRELSecurityGroup.GroupId
+          - !GetAtt SSHRELSecurityGroup.GroupId
```
Comment (delgod) [inline IaC/rel.cd/JenkinsStack.yml:464]: 1. in this template in the first line I see `us-west-1` region, but stack is run in `eu-west-1`
2. line 540 has version `2.222.3` but in fact, on web interface I see `2.235.5`

please fix line 1 and line 540

**Review by delgod — CHANGES_REQUESTED** [review]

File: IaC/rel.cd/JenkinsStack.yml (hunk 1 above)
Comment (Sudokamikaze) [reply IaC/rel.cd/JenkinsStack.yml:464]: 1. Working on it
2. Fixed locally

Comment (Sudokamikaze) [reply IaC/rel.cd/JenkinsStack.yml:464]: Regarding second point, I suggest to create another PR where it'll be fixed

@delgod

Comment (delgod) [reply IaC/rel.cd/JenkinsStack.yml:464]: Absolutely we can have another PR for these fixed

**Review by delgod — APPROVED** [review]

File: IaC/rel.cd/JenkinsStack.yml (hunk 1 above)
Comment (Sudokamikaze) [reply IaC/rel.cd/JenkinsStack.yml:464]: In latest rebase: 0b6f1f3 
Updated first-line comment with proper zone

**Review by delgod — APPROVED** [review]

**Pull request merged**